tests/tools.vitruv.applications.transitivechange.tests/META-INF/MANIFEST.MF

```diff
@@ -16,7 +16,8 @@ Require-Bundle: org.junit.jupiter.api,
  org.hamcrest.core,
  edu.kit.ipd.sdq.activextendannotations,
  org.apache.log4j,
- org.eclipse.xtend.lib
+ org.eclipse.xtend.lib,
+ tools.vitruv.applications.testutility;bundle-version="0.3.0"
```

Comment:
```suggestion
 tools.vitruv.applications.testutility
```

Reply:
Removed in 49314c9ac45c88cd3beab359de9bd55734b9656a